index.js

```diff
@@ -244,12 +254,15 @@ Use copy to do the following:
 🌟 EXAMPLE of return: ["Amedeo Modigliani", "Vasiliy Kandinskiy", "Diego Rivera"....]
 */
 
-function listOfNames(/*Your Code Here*/) {
-  /*Your Code Here*/
+function listOfNames(array) {
```

Comment:
Nice way of using the spread operator (...) for the copy. Neat way of changing to just the name with the splice function.